### DISCUSSION TITLE
My F1Score is 0 and pixelRUROC also 0? Why?

### DISCUSSION THREAD

**Discussion by lofangmeng** [open-edge-platform/anomalib#1271] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/75239990/210983509-2adf825e-48e2-4b78-a9eb-9561957d6a08.png)
My custom datasets was prepared same like the MvTec Format
![image](https://user-images.githubusercontent.com/75239990/210984514-640dfa0d-ad19-4c63-9f0c-a3795f478ab6.png)
but the result come out with no ground-truth and predict
![image](https://user-images.githubusercontent.com/75239990/210985243-a8690c43-38b7-4745-826b-4856f3a219e1.png)
Is that any wrong on my datasets or what?

**Comment by samet-akcay** [comment]
@lofangmeng, could you please share your config file for reference? Thanks?

**Comment by lofangmeng** [comment]
dataset:
  name: mvtec #options: [mvtec, btech, folder]
  format: mvtec
  path: ./datasets/MVTec
  task: segmentation
  category: makoutie
  image_size: 224
  train_batch_size: 32
  test_batch_size: 32
  num_workers: 0
  transform_config:
    train: null
    val: null
  create_validation_set: false
  tiling:
    apply: false
    tile_size: null
    stride: null
    remove_border_count: 0
    use_random_tiling: False
    random_tile_count: 16

model:
  name: patchcore
  backbone: wide_resnet50_2
  pre_trained: true
  layers:
    - layer2
    - layer3
  coreset_sampling_ratio: 0.1
  num_neighbors: 9
  normalization_method: min_max # options: [null, min_max, cdf]

**Comment by lofangmeng** [comment]
i was using the MVTec format

**Comment by lofangmeng** [comment]
are the ground trouh should be white color? cannot be other color?

**Comment by samet-akcay** [comment, marked as answer]
> are the ground trouh should be white color? cannot be other color?

the datamodule would expect a binary mask of 1's and 0's, where 1 (white) would be the anomalous region. You could refer to an MVTec ground-truth image.

**Comment by ashwinvaidya17** [comment]
@lofangmeng were you able to get the results? If possible can you share an example of the ground truth masks?
